src/OpenTelemetry/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 * Default `Resource` will now contain service.name instead of Telemetry SDK.
   [#1744](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1744)
+* Exporters fall back to the default `Resource`'s `service.name` value instead
```

Comment:
Exporters have a diff. changelog, so this must be added there instead.
For OpenTelemetry, the only thing in changelog is just the following line 
"Added GetDefaultResource() method to Provider."